operations/restrictions.md

```diff
@@ -369,3 +375,113 @@ Removes restrictions from the service.
+         "BaseRateId": "e5b538b1-36e6-43a0-9f5c-103204c7f68e",
+         "Days": {
+            "Monday": false,
+            "Tuesday": false,
+            "Wednesday": false,
+            "Thursday": false,
+            "Friday": true,
+            "Saturday": true,
+            "Sunday": true
+         },
+         "MinAdvance": "P0Y0M1DT0H0M0S",
+         "MaxAdvance": "P0Y0M3DT0H0M0S"
+      }
+   ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) restrictions will be set in. |
+| `Data` | array of [RestrictionSetData](#restriction-set-data) | required | Parameters of restrictions. |
+
+#### Restriction set data
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Identifier` | string | optional | Identifier of the restriction within the transaction. |
+| `ExternalIdentifier` | string | optional | External identifier of the restriction. |
+| `Type` | string | required | [Restriction type](#restriction-type). |
+| `ExactRateId` | string | optional | Unique identifier of the restricted exact [Rate](rates.md#rate). |
+| `BaseRateId` | string | optional | Unique identifier of the restricted base [Rate](rates.md#rate). |
+| `RateGroupId` | string | optional | Unique identifier of the restricted [Rate group](rates.md#rate-group). |
+| `ResourceCategoryId` | string | optional | Unique identifier of the restricted [Resource category](resources.md#resource-category). |
+| `ResourceCategoryType` | string | optional | Name of the restricted [Resource category type](resources.md#resource-category-type). |
+| `StartUtc` | string | optional | Start of the restricted interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | optional | End of the restricted interval in UTC timezone in ISO 8601 format. |
+| `Days` | [DaysParameters](#days-parameters) | required | The restricted days of week. |
+| `MinAdvance` | string | optional | The minimum time before the reservation starts, you can reserve in ISO 8601 duration format. |
+| `MaxAdvance` | string | optional | The maximum time before the reservation starts, you can reserve in ISO 8601 duration format. |
+| `MinLength` | string | optional | Minimal reservation length in ISO 8601 duration format. |
+| `MaxLength` | string | optional | Maximal reservation length in ISO 8601 duration format. |
+| `MinPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the minimum price per time unit. |
+| `MaxPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the maximum price per time unit. |
+
+#### Days parameters
+| Property | Type | Contract | Description |
```

Comment:
Are they all required? Wouldn't it make sense to default to false it it's not provided same as we have with extents?

Reply:
It's the same DTO that is being used in Commander ... I'd say it makes the object consistent without the need to handle nulls...